Feature/geolocation (#267)
infinum/eightshift-forms: src/Hooks/Variables.md

## Set forms to develop mode

> This variable will set forms to develop mode that will do some changes in code.

Reviewer: _some changes_

What changes? 👀

Author: is is listed in the next paragraph "actions"

Reviewer: I'd add then:

> This variable will set forms to develop mode, which will do the code changes listed in the 'actions' paragraph below.

You **must** add both constants in order to use Google reCaptcha.

Type:
* string

Default:
* empty

```php
define('ES_GOOGLE_RECAPTCHA_SITE_KEY', '<site-key>');
define('ES_GOOGLE_RECAPTCHA_SECRET_KEY', '<secret-key>');
```

## Set geolocation use

This constant will manually set geolocation usage to active. While this constant is active you will not be able to deactivate it from settings.

Type:
* boolean

Default:
* false

```php
define('ES_GEOLOCATION_USE', true);
```

## Set geolocation WP-Rocket advanced cache

This constant will activate WP-Rocket advanced cache filter to create custom geolocation function in advanced-cache.php file.

Type:
* boolean

Default:
* false

```php
define('ES_GEOLOCATION_USE_WP_ROCKET_ADVANCED_CACHE', true);
```

## Set geolocation IP

This constant will manually set geolocation IP address to a fixed value and the geolocation will always return the same location.

Type:
* string

Default:
* empty

```php
define('ES_GEOLOCATION_IP', '<ip>');
```

## Set geolocation cookie name

> This constant will define cookie name used for geolocation. In general you will never need to change this value.

Reviewer: ```suggestion
This constant will define the cookie name used for geolocation. Generally, you will never need to change this value.
```

Type:
* string

Default:
* esForms-country

```php
define('ES_GEOLOCATION_COOKIE_NAME', '<cookie-name>');
```

## Set geolocation Phar location

> This constant will define geolocation phar location. With this constant you can provide your custom projects geolocation phar file. You will use this if you want to control the phar file version and keep it better up to date.

Reviewer: ```suggestion
This constant will define geolocation phar location. With this constant, you can provide your custom project's geolocation phar file. You will use this if you want to control the phar file version and keep it better up to date.
```

Type:
* string

Default:
* absolute path the to the Eightshift-form geolocation .phar file.

```php
define('ES_GEOLOCATION_PHAR_PATH', '<absolute-path>');
```

## Set geolocation Db location

> This constant will define geolocation db location. With this constant you can provide your custom projects geolocation db file. You will use this if you want to control the db file version and keep it better up to date.

Reviewer: ```suggestion
This constant will define geolocation db location. With this constant, you can provide your custom project's geolocation db file. You will use this if you want to control the db file version and keep it better up to date.
```
